skin/platform-linux.css

```diff
@@ -1,4 +1,6 @@
-/*Only Linux Styles*/
+/**
+ * Linux style adaptations
+ */
```

Comment:
I don't see why this comment needed to change.  Please revert this change (and the other two like it).